apps/web/app/(app)/environments/[environmentId]/settings/members/components/EditMemberships/MemberActions.tsx

```diff
@@ -74,8 +74,9 @@ export default function MemberActions({ team, member, invite, showDeleteButton }
   const handleShareInvite = async () => {
     try {
       if (!invite) return;
-
-      const { inviteToken } = await createInviteTokenAction(invite.id);
+      const newInviteToken = await createInviteTokenAction(invite.id);
+      if (!newInviteToken) return;
```

Comment:
@Dhruwang what was your idea behind this? when do you expect the invite token to be empty?